.gitlab-ci.yml

```diff
@@ -13,7 +13,8 @@ test:
     - kaolin
   stage: test
   script:
-    - flake8 --config=.flake8 .
     - source setenv.sh
     - python setup.py develop
-    - pytest --cov=kaolin/ tests/
+    - ./tests/examples/run/test_classification.sh
+      #- flake8 --config=.flake8 .
+      #- pytest --cov=kaolin/ tests/
```

Comment:
Can delete

Reply:
I uncommented and moved it back where it were

Reply:
Ah yes, my bad :)